$NetBSD: patch-ac,v 1.4 2010/05/26 01:32:33 obache Exp $

* fixes build failure:
LYMain.c: In function 'SetLocale':
LYMain.c:941: error: 'LC_ALL' undeclared (first use in this function)
LYMain.c:941: error: (Each undeclared identifier is reported only once
LYMain.c:941: error: for each function it appears in.)

* fixes PR#43340

--- src/LYMain.c.orig	2009-07-05 23:52:33.000000000 +0000
+++ src/LYMain.c
@@ -57,7 +57,7 @@
 #include <io.h>
 #endif
 
-#if defined(LOCALE) && !defined(HAVE_LIBINTL_H)
+#if defined(LOCALE)
 #undef gettext			/* Solaris locale.h prototypes gettext() */
 #include <locale.h>
 #ifndef HAVE_GETTEXT
@@ -3079,7 +3079,7 @@ G)oto's" },
 		if (first) {
 		    printf("Other restrictions (see the user's guide):\n");
 		}
-		value = show_restriction(table[j].name);
+		value = show_restriction(name);
 		printf("%s%s (%s)", column ? ", " : "  ", name, value);
 		column += 5 + strlen(name) + strlen(value);
 		if (column > 50) {
